src/instruments/src/EWD/elements/PseudoFWC.tsx

```diff
@@ -0,0 +1,1597 @@
+                memoInhibit: !!(tomemo === 1 || ldgmemo === 1),
+                failure: 0,
+                sysPage: -1,
+                side: 'LEFT',
+            },
+        '0000105': // OUTR TK FUEL XFRD
+            {
+                flightPhaseInhib: [], // Plus check that outer tanks not empty
+                simVarIsActive: !!(leftOuterInnerValve || rightOuterInnerValve),
+                whichCodeToReturn: [0],
+                codesToReturn: ['000010501'], // config memo
+                memoInhibit: !!(tomemo === 1 || ldgmemo === 1),
+                failure: 0,
+                sysPage: -1,
+                side: 'LEFT',
+            },
+        '0000110': // FOB BELOW 3 T or 6600 LBS
+            {
+                flightPhaseInhib: [],
+                simVarIsActive: fobRounded < 3000,
+                whichCodeToReturn: [unit === '1' ? 0 : 1],
+                codesToReturn: ['000011001', '0000011002'], // config memo
+                memoInhibit: !!(tomemo === 1 || ldgmemo === 1),
+                failure: 0,
+                sysPage: -1,
+                side: 'LEFT',
+            },
+        '0000305': // GPWS FLAP MODE OFF
+            {
+                flightPhaseInhib: [],
+                simVarIsActive: !!gpwsFlapMode,
+                whichCodeToReturn: [0],
+                codesToReturn: ['000030501'], // Not inhibited
+                memoInhibit: !!(tomemo === 1 || ldgmemo === 1),
+                failure: 0,
+                sysPage: -1,
+                side: 'LEFT',
+            },
+        '0000140': // T.O. INHIBIT
+            {
+                flightPhaseInhib: [1, 2, 6, 7, 8, 9, 10],
+                simVarIsActive: showTakeoffInhibit,
+                whichCodeToReturn: [0],
+                codesToReturn: ['000014001'],
+                memoInhibit: false,
+                failure: 0,
+                sysPage: -1,
+                side: 'RIGHT',
+            },
+        '0000150': // LDG INHIBIT
+            {
+                flightPhaseInhib: [1, 2, 3, 4, 5, 6, 9, 10],
+                simVarIsActive: showLandingInhibit,
+                whichCodeToReturn: [0],
+                codesToReturn: ['000015001'],
+                memoInhibit: false,
+                failure: 0,
+                sysPage: -1,
+                side: 'RIGHT',
+            },
+        '0000350': // LAND ASAP RED
+            {
+                flightPhaseInhib: [],
+                simVarIsActive: !!landASAPRed,
+                whichCodeToReturn: [0],
+                codesToReturn: ['000035001'],
+                memoInhibit: false,
+                failure: 0,
+                sysPage: -1,
+                side: 'RIGHT',
+            },
+        '0000360': // LAND ASAP AMBER
+            {
+                flightPhaseInhib: [],
+                simVarIsActive: !!(!landASAPRed && !onGround && (
+                    engine1State === 0
+                    || engine2State === 0
+                    || fobRounded < 3000
```

Comment:
This is obviously a bit simplified, but I think it's fine for now (should be related to FUEL L+R WING TK LO LEVEL).